test/math/QuaternionTest.ooc

```diff
@@ -86,7 +85,7 @@ QuaternionTest: class extends Fixture {
 			expect(conjugate z == -quaternion5 z)
 		})
 		this add("normalized", func {
-			normalized := quaternion0 normalized
+			normalized := quaternion0 normalized as Quaternion
```

Comment:
Yes, I know. I think the problem is in the `expect` function here. The explicit cast in the property is not needed for general usage, I think it was basically there because it fixed the test.